cleanlab_studio/utils/databricks_utils.py

```diff
@@ -0,0 +1,88 @@
+import os
+import zipfile
+from datetime import datetime
+
+
+def create_path_based_imageset_archive(folder_path, archive_name=None) -> str:
+    """
+    Archives an imageset stored on Databricks which can then be uploaded Cleanlab Studio.
+    The imageset folder should match the layout described in the Cleanlab Studio documentations.
+    https://help.cleanlab.ai/guide/concepts/datasets/#image
+
+    Args:
+        folder_path: The POSIX-style path to your imageset.
+        archive_name: A name you choose for the imageset archive.
+
+    Returns:
+        The relative path to the archived imageset.
+    """
+    folder_name = os.path.basename(os.path.normpath(folder_path))
```

Comment:
somewhat of a nit, but we generally prefer using `pathlib` over `os`. `pathlib` is the more modern library. if it's not too much effort, can you rewrite these utils using `pathlib`?